### ISSUE TITLE
(SENet) Squeeze-and-Excitation Networks

### ISSUE THREAD

**Issue by yutayamazaki** [yutayamazaki/paper_reading#5]
### 論文
Squeeze-and-Excitation Networks
https://arxiv.org/pdf/1709.01507.pdf

### 著者・所属機関
Jie Hu, Li Shen, Samuel Albanie, Gang Sun, Enhua Wu

### 投稿日
2018/10/25

## Abstract
- チャンネル間の関係性を考慮したSE-blockの提案

## 先行研究と比べて何がすごい？
- ResNetなど既存のCNNに簡単に追加できる
- 計算コストもあまりかからない

## 技術や手法の肝はどこ？
- チャンネルごとの平均値(GAPの結果)をDense(ReLU)->Dense(Sigmoid)につないで学習
- つまり，通常のCNNでは考慮しない，チャンネル間の重みを学習する機構が入った
- それをsqueeze and excitation blockという
- channel間の重みを学習する際は，GAPの情報を使うのがいいことが実験的にわかった

## どうやって有効だと検証した？
- ILSVRC2017 first place

## 次に読むべき論文は？
- Faster R-CNN: Towards Real-Time Object Detection with Region Proposal Networks, https://arxiv.org/abs/1506.01497
- Dual Path Networks, https://arxiv.org/abs/1707.01629
- Densely Connected Convolutional Networks, https://arxiv.org/abs/1608.06993
- Deep Roots: Improving CNN Efficiency with Hierarchical Filter Groups, https://arxiv.org/abs/1605.06489
- Genetic CNN, https://arxiv.org/abs/1703.01513
- Progressive Neural Architecture Search, https://arxiv.org/abs/1712.00559
- SMASH: One-Shot Model Architecture Search through HyperNetworks, https://arxiv.org/abs/1708.05344
- Accelerating Neural Architecture Search using Performance Prediction, https://arxiv.org/abs/1705.10823
- Designing Neural Network Architectures using Reinforcement Learning, https://arxiv.org/abs/1611.02167
- Efficient Neural Architecture Search via Parameter Sharing, https://arxiv.org/abs/1802.03268
- Attention Is All You Need, https://arxiv.org/abs/1706.03762
- SCA-CNN: Spatial and Channel-wise Attention in Convolutional Networks for Image Captioning, https://arxiv.org/abs/1611.05594
- CBAM: Convolutional Block Attention Module, https://arxiv.org/abs/1807.06521
- Residual Attention Network for Image Classification, https://arxiv.org/abs/1704.06904
- ShuffleNet: An Extremely Efficient Convolutional Neural Network for Mobile Devices, https://arxiv.org/abs/1707.01083
- Shake-Shake regularization, https://arxiv.org/abs/1705.07485
- Deep Pyramidal Residual Networks, https://arxiv.org/abs/1610.02915
- Exploring the Limits of Weakly Supervised Pretraining, https://arxiv.org/abs/1805.00932
